src/core/function/config.js

```diff
@@ -49,23 +52,33 @@ export function configFactory (config, emit) {
    */
   function _config (options) {
     if (options) {
-      const prev = mapObject(config, clone)
+      if (options.epsilon !== undefined) {
+        // backwards compatibility
+        console.warn('Warning: The configuration option "epsilon" is deprecated. Use "relTol" and "absTol" instead.')
+        const optionsFix = mapObject(options, clone)
+        optionsFix.relTol = options.epsilon
+        optionsFix.absTol = options.epsilon * 1e-3
+        delete optionsFix.epsilon
+        return _config(optionsFix)
+      } else {
```

Comment:
Can you remove the `else` case, so we keep the "old" as is and only add a small `if() {...}` section for backward compatibility that we can remove some day without having to alter the rest of the code?

Reply:
Thanks, got it. I thougt it could look more common with the else. But I guess the diff will look cleaner in the future.